impala/tests/test_data_types.py

```diff
@@ -54,43 +54,47 @@ def date_table(cur):
     table_name = 'tmp_date_table'
     ddl = """CREATE TABLE {0} (d date)""".format(table_name)
     cur.execute(ddl)
+    cur.execute('''insert into {0}
+                   values (date "0001-01-01"), (date "1999-9-9")'''.format(table_name))
     try:
         yield table_name
     finally:
         cur.execute("DROP TABLE {0}".format(table_name))
 
-
-@pytest.mark.connect
-def test_date_basic(cur, date_table):
+def setup_test_date(cur, date_table):
```

Comment:
The naming doesn't make sense to me - this function does a select and checks the result, so it is not a setup function.